source/CommonJobs/CommonJobs.MVC.UI/Areas/EvalForm/Controllers/EvalFormController.cs

```diff
@@ -0,0 +1,30 @@
+﻿using CommonJobs.Application.MyMenu;
+using CommonJobs.Domain;
+using CommonJobs.Domain.MyMenu;
+using CommonJobs.Infrastructure.Mvc;
+using CommonJobs.Mvc.UI.Infrastructure;
+using NLog;
+using Raven.Client.Linq;
+using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Web;
+using System.Web.Mvc;
+using CommonJobs.Utilities;
+
+namespace CommonJobs.Mvc.UI.Areas.EvalForm
+{
+    [CommonJobsAuthorize]
+    //TODO: Add documentation
+    [Documentation("docs/manual-de-usuario/...")]
```

Comment:
Maybe it is a good idea to create a draft documentation file and reference it here (it is easy to forgot).